qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -74,11 +76,125 @@ def get_raw(self) -> np.ndarray:
+        interval for accurate results.
+
+        Raises:
+            RuntimeError: If no instrument attached to Parameter.
+        """
+        if self.instrument is None:
+            raise RuntimeError("No instrument attached to Parameter.")
+
+        dt = self.instrument.dt()
+        nplc = self.instrument.nplc()
+        plc = 1/float(self.instrument.ask('localnode.linefreq'))
+        if nplc * plc > dt:
+            warnings.warn(f'Integration time of {nplc*plc*1000:.1f} ' +
+                          f'ms is longer than {dt*1000:.1f} ms set ' +
+                          'as measurement interval. Consider lowering ' +
+                          'NPLC or increasing interval.', UserWarning, 2)
+
+    def _set_mode(self, mode: str) -> None:
+        """
+        A helper function to set correct units and labels.
+
+        Args:
+            mode: User defined mode for the timetrace. It can be either
+            current, "i", or voltage "v".
+        """
+        if mode == 'i':
+            self.unit='A'
+            self.label='Current'
+        if mode == 'v':
+            self.unit='V'
+            self.label='Voltage'
+
+    def _time_trace(self, npts: int, dt: float, mode: str) -> np.ndarray:
+        """
+        The function that prepares a Lua script for timetrace data acquisition.
+
+        Args:
+            npts: Number of points.
+            dt: Infinitesimal time resolution.
+            mode: User defined mode for the timetrace. It can be either
+            current, "i", or voltage "v".
+
+        Raises:
+            RuntimeError: If no instrument attached to Parameter.
+        """
+
+        if self.instrument is None:
+            raise RuntimeError("No instrument attached to Parameter.")
+
+        channel = self.instrument.channel
+        self.shape = (npts,)
+
+        script = ['{}.measure.count={}'.format(channel, self.shape[0]),
+        'oldint={}.measure.interval'.format(channel),
+        '{}.measure.interval={}'.format(channel, dt),
+        '{}.nvbuffer1.clear()'.format(channel),
+        '{}.measure.{}({}.nvbuffer1)'.format(channel, mode, channel),
+        '{}.measure.interval=oldint'.format(channel),
+        '{}.measure.count=1'.format(channel),
+        'format.data = format.REAL32',
+        'format.byteorder = format.LITTLEENDIAN',
+        'printbuffer(1, {}, {}.nvbuffer1.readings)'.format(self.shape[0], channel)]
+
+        return self.instrument._execute_lua(script, npts)
+
+    def get_raw(self) -> np.ndarray:  # pylint: disable=E0202
+
+        if self.instrument is None:
+            raise RuntimeError("No instrument attached to Parameter.")
+
+        mode = self.instrument.timetrace_mode()
+        self._check_time_trace()
+        npts = self.instrument.npts()
+        dt = self.instrument.dt()
+        data = self._time_trace(npts, dt, mode)
+        return data
+
+
+class TimeAxis(Parameter):
+    """
+    A simple :class:`.Parameter` that holds the time interval
```

Comment:
Perhaps it's just me, but I find the wording confusing. The `TimeAxis` doesn't hold the time interval, it holds all the times (relative to the measurement start) at which the points of the `TimeTrace` were acquired.